src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -269,7 +269,19 @@ public void dispose() {
 
     @Nullable
     public FlutterSdk getFlutterSdk() {
-      return FlutterSdk.forPath(myPeer.getSdkComboPath());
+      final String sdkPath = myPeer.getSdkComboPath();
+
+      //Ensure the local filesystem has caught up to external processes (e.g., git clone).
+      if (!sdkPath.isEmpty()) {
+        try {
+          LocalFileSystem
+            .getInstance().refreshAndFindFileByPath(sdkPath);
```

Comment:
I tried this a few different ways but couldn't get the threading right.  The good news is that this method is only accessed in this wizard step so it's effectively the same as calling it after `flutter precache`.